src/utils.ts

```diff
@@ -535,25 +537,34 @@ export async function getEnsTextRecord(
   options: any = {}
 ) {
   const {
-    ensResolvers: ensResolversOpt,
+    ensResolvers: ensResolversOpt = networks[network]?.ensResolvers ||
```

Comment:
Where do we use this `ensResolverOpts` ? Seems not to be used anywhere on snapshot repo

Reply:
Removed it